frontend/amundsen_application/static/js/interfaces/TableMetadata.ts

```diff
@@ -33,6 +33,10 @@ export interface TableSource {
   source_type: string;
 }
 
+export interface TableDocumentation {
```

Comment:
FYI, we have a feature called programmatic description which you could link external table metadata with markdown syntax (https://technology.edmunds.com/2020/05/27/Adding-Data-Quality-into-Amundsen-with-Programmatic-Descriptions/) . Do you think that will cover your use case?

Reply:
Oh wow yes this is exactly what I need. I must've missed this model. I do like the UX of the buttons in the top right and do see a Documentation column existing there but since this functionality will suit my needs I will close this for now!